CoE_jump_start/requirements.txt

```diff
@@ -26,4 +26,6 @@ pillow_heif==0.16.0
 ipython==8.26.0
 tqdm==4.66.3
 matplotlib==3.8.0
-seaborn==0.12.2
\ No newline at end of file
+seaborn==0.12.2
```

Comment:
It's always preferable to follow the alphabetical order for every block of dependencies. I will add this to our guidelines on how to sort the dependencies.